### DISCUSSION TITLE
[Feature] Resumable chunked upload option for large files upload (>100 MB) over the internet

### DISCUSSION THREAD

**Discussion by arpanghosh8453** [immich-app/immich#6524] [Feature Request]
### The feature

I don't know if it's too much to ask, but I will leave my idea here. 

I am using Cloudflare tunnels and cannot upload files larger than 100 MB because of [single chunk upload limitation ](https://community.cloudflare.com/t/maximum-upload-size-is-limit/418490). The same issue existed on the [Filebrowser](https://filebrowser.org/) service, but they [fixed it here](https://github.com/filebrowser/filebrowser/pull/2145) using chunked upload. That also helps in upload resuming for interrupted transfers. 

Here is how their UI looks like for the option:

![image](https://github.com/immich-app/immich/assets/26424944/0aac0cd1-d9a9-4cc1-88c3-886b50c8c309)

I am not sure how immich handles upload, or how difficult it is to implement. This sounds like a very solid feature to be regardless of the Cloudflare chunk limit bypass. I hope people proxying their service through Cloudflare will agree on this fact. I myself can use VPN or direct local transfer, but my parents living abroad are not tech savvy and cannot upload files to immich ( video files larger then 100 MB ). Please let me know your thoughts :) 

And thank you everyone for your time. I LOVE IMMICH!!!

### Platform

- [X] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #1674
